pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "amplipi"
-version = "0.2.1"
+version = "0.2.1+fc5f368-lms_metadata_2-dirty"
```

Comment:
Looks like the deploy script left this change here. This file should not be changed.